librz/arch/asm.c

```diff
@@ -1861,6 +1861,18 @@ rz_asm_colorize_asm_str(RZ_BORROW RzStrBuf *asm_str, RZ_BORROW RzPrint *p, RZ_NU
 	return colored_asm;
 }
 
+/**
+ * \brief Free a RzAsmParseParam
+ *
+ * \param p The parameter struct.
+ */
+RZ_API void rz_asm_parse_param_free(RZ_OWN RzAsmParseParam *p) {
+	if (!p) {
+		return;
+	}
+	free(p);
+}
```

Comment:
maybe just have in the headers `#define rz_asm_parse_param_free(x) free(x)`

Reply:
Decided against it because it breaks reverse search for clangd (at least in my case).
But thanks for the hint. Found two leaks this way.